presto-server/pom.xml

```diff
@@ -31,7 +31,7 @@
             <artifactId>presto-main</artifactId>
         </dependency>
 
-        <!-- The scope of all of the following dependencies are provided to avoid them being added to lib -->
+        <!-- The scope of all of the following dependencies (except Kafka) are provided to avoid them being added to lib -->
```

Comment:
why are we making these changes? this is for presto-server, better not to pollute the lib